src/archivist/filter/index.js

```diff
@@ -20,6 +20,15 @@ const { CiceroMarkTransformer } = ciceroMark;
 
 const ciceroMarkTransformer = new CiceroMarkTransformer();
 
+/**
+ * Filter document content and convert it to Markdown
+ *
+ * @param {Object} params - Filter parameters
+ * @param {string|Buffer} params.content - Content to filter: a buffer containing PDF data in case mimetype associated is PDF or a DOM dump of an HTML page given as a string
+ * @param {string} params.mimeType - Mime type of the given content
+ * @param {string} params.documentDeclaration - @see {@link ./src/archivist/services/documentDeclaration.js}
```

Comment:
I understand [`@see`](https://jsdoc.app/tags-see.html) as being a first-level tag.
```suggestion
 * @param {string} params.documentDeclaration - see {@link ./src/archivist/services/documentDeclaration.js}
```